### DISCUSSION TITLE
How do I set project wide `PYTHONPATH` for this extension?

### DISCUSSION THREAD

**Discussion by tigerinus** [microsoft/vscode-pylint#88] [Q&A]
Here is my current project folder structure

```
.
├── api
├── integration_tests
├── k8s
├── src
├── unit_tests
└── venv
```

All python code are in the `src` folder, including internally dependent modules.

In command line, I run `PYTHONPATH=src pylint ./src` all the time to lint check the code.

Without `PYTHONPATH=src`, pylint would give errors like

![image](https://user-images.githubusercontent.com/7172560/168198377-33b4c762-a81a-4185-945c-05bfa8405b9f.png)

How do I pass `PYTHONPATH` to this pylint extension?

**Comment by karthiknadig** [comment]
@tigerinus Can you try this and see if it works? This is for windows
```
> set PYTHONPATH=src
> code .
```

**Reply by tigerinus** [reply]
I am on Windows, but I prefer launching VSCode from Start menu.

Plus that I use VSCode + WSL2 only.

So this wouldn't be an option.
